pkg/cmd/server/api/types.go

```diff
@@ -235,6 +235,11 @@ type ServingInfo struct {
 	ServerCert CertInfo
 	// ClientCA is the certificate bundle for all the signers that you'll recognize for incoming client certificates
 	ClientCA string
+	// MaxRequestsInFlight is the number of concurrent requests allowed to the server. If zero, no limit.
```

Comment:
The alternative is to create another top level struct for them for each section that needs them.  Most Go things *would* support this, and probably should.